please check that file 
     [java]  - with linked exception:
     [java] [org.xml.sax.SAXParseException; systemId: file:/home/vsts/work/1/s/target/test-definitions/cmpSqlResults-config.xml; lineNumber: 74; columnNumber: 17; The end-tag for element type "datasource" must end with a '>' delimiter.]
     [java] [ERROR] 2021-02-01 11:21:45,209 RunTests - Errors while reading configuration file: test-definitions/cmpSqlResults-config.xml


https://dev.azure.com/DBTestCompare/Build/_build/results?buildId=177&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=afd94d31-0f2b-5cc6-5fec-a52e557ade2b

--- a/test-definitions/cmpSqlResults-config.xml
+++ b/test-definitions/cmpSqlResults-config.xml
@@ -64,6 +64,7 @@
             <url>jdbc:postgresql://${POSTGRESSSQLDBSERVER}:5432/${POSTGRESSSQLDBNAME}</url>
             <user>${USERNAME}</user>
             <password>${PASSWORD}</password>
+		</datasource>            
         <datasource maxPollSize="6"> 
             <name>SNOWFLAKE_SERVER</name>
             <driver>com.snowflake.client.jdbc.SnowflakeDriver</driver>

